bitshares/message.py

```
@@ -170,4 +170,10 @@ def verify(self, **kwargs):
         if format(pk, self.blockchain.prefix) != memo_key:
             raise InvalidMessageSignature
 
+        if "account" in kwargs:
+            return kwargs["account"] == account_name
```

can you raise a new Exception when accounts do not match but provided in kwargs?